Lesson9/Home_task_9.py

```diff
@@ -2,47 +2,49 @@
 
 class Medicine:
 
-    method_of_administration = "inside"
+    method_of_administration_default = "inside"
     medicine_count = 0
 
-    def __init__(self, name, dosage, form, method_of_administration):
+    def __init__(self, name, dosage, form, method_of_administration=None):
         self.name = name
         self.dosage = dosage
         self.form = form
+        self.method_of_administration = Medicine.method_of_administration_default if method_of_administration == None else method_of_administration
```

Comment:
self.method_of_administration = Medicine.method_of_administration_default if method_of_administration is None else method_of_administration